app/modules/wallet/addresses/addresses.controller.ts

```diff
@@ -0,0 +1,56 @@
+import * as restify from 'restify';
+import { InternalServerError, NotFoundError } from 'restify-errors';
+import IController from '../../../interfaces/utils/IController';
+import IRequest from '../../../interfaces/utils/IServer';
+import WalletAddress from '../../../models/wallet_address.model';
+import { IWallet } from '../../../interfaces/models';
+
+export default class WalletsController implements IController {
+  public async put(req: IRequest, res: restify.Response, next: restify.Next) {
+    try {
+      const walletAddress = await WalletAddress.create({
```

Comment:
You might first want to check if the `wallet_id` belongs to the `client_id` which is requesting to create address in that wallet. Also, the endpoint i think will be POST instead of PUT. As PUT is for updation? 
`POST /wallets/:id/addresses`

Reply:
Also, use typescript's types frequently in your code so we make use of that. e.g
```javascript
const walletAddress = <IWalletAddress> await WalletAddress.create(<IWalletAddress> {...});
```

Reply:
Kept it PUT as it was an updation of `wallet` entity. I'll change it to POST, to keep it consistent with row creation.